File: app/src/main/java/com/ravisharma/playbackmusic/database/ShuffleSongsDao.java
package com.ravisharma.playbackmusic.database;

import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.Query;

import com.ravisharma.playbackmusic.database.model.ShuffleSongs;

import java.util.List;

@Dao
public interface ShuffleSongsDao {

    @Insert
    void addSong(ShuffleSongs songs);

    @Query("DELETE FROM shuffledSongs WHERE id=:id")
    void deleteFromShuffle(long id);

    @Query("DELETE FROM shuffledSongs")
    void removeAll();

    @Query("SELECT * FROM shuffledSongs")
    List<ShuffleSongs> getShuffleSongs();
}

File: app/src/main/java/com/ravisharma/playbackmusic/database/dao/ShuffleSongsDao.kt
package com.ravisharma.playbackmusic.database.dao

import androidx.room.Dao
import androidx.room.Insert
import androidx.room.Query
import com.ravisharma.playbackmusic.database.model.ShuffleSongs

@Dao
interface ShuffleSongsDao {
    @Insert
    fun addSong(songs: ShuffleSongs)

    @Query("DELETE FROM shuffledSongs WHERE id=:id")
    fun deleteFromShuffle(id: Long)

    @Query("DELETE FROM shuffledSongs")
    fun removeAll() : Int

    @get:Query("SELECT * FROM shuffledSongs")
    val shuffleSongs: List<ShuffleSongs>
}